src/components/ViewItem/ViewItem.js

```diff
@@ -111,6 +111,7 @@ class ViewItem extends React.Component {
   }
 
   hideChangeDueDateDialog() {
+    this.refreshLoans();
```

Comment:
This one had already been here, and it was removed in https://github.com/folio-org/ui-checkout/pull/447 for 
https://issues.folio.org/browse/UICHKOUT-597

Are you sure we introduce no regression here?

In https://github.com/folio-org/ui-checkout/pull/447 I do not see any added tests to guard the bug. So It's probably not auto-tested.

Reply:
That was done a while ago, and I can't remember the details of why I removed the call to `refreshLoans()`. So I can't say that you _shouldn't_ add it back in now. But please confirm that the bug described in https://issues.folio.org/browse/UICHKOUT-597 does not regress when you do. I apologize for the lack of a test for that.